hourly_watch.py

```diff
@@ -15,6 +15,50 @@
 
 
 # Ska-specific watchers
+class SkaURLWatch(JobWatch):
+    def __init__(self, task, maxage_hours, url=None,):
+        self.type = 'URL'
+        self.basename = url
+        super(SkaURLWatch, self).__init__(task, url, maxage=maxage_hours * HOURS)
+
+    @property
+    def headers(self):
+        if not hasattr(self, '_headers'):
+            try:
+                import urllib2
+                response = urllib2.urlopen(self.basename)
+                self._headers = response.headers
+            except:
+                self._exists = False
+                self._headers = None
+        return self._headers
+
+    @property
+    def exists(self):
+        if not hasattr(self, '_exists'):
+            self._exists = self.headers is not None
+        return self._exists
+
+    @property
+    def age(self):
+        if not hasattr(self, '_age'):
+            if self.headers is not None:
+                import time
```

Comment:
Only use these local imports to avoid circular imports or for modules that are typically used interactively and it makes a difference.  Otherwise it is easier to have the imports at the top.

(I've been schooled on this a bit by astropy, esp. Erik Bray, so past code might not adhere).